modules/UI/videolayout/RemoteVideo.js

```diff
@@ -572,26 +581,25 @@ RemoteVideo.prototype.updateView = function () {
  * Updates the UI to reflect user's connectivity status.
  */
 RemoteVideo.prototype.updateConnectionStatusIndicator = function () {
-    const isActive = this.isConnectionActive();
-
-    if (isActive === null) {
-        // Cancel processing at this point - no update
-        return;
-    }
+    const connectionStatus = this.user.getConnectionStatus();
 
-    logger.debug(this.id + " thumbnail is connection active ? " + isActive);
+    logger.debug(`${this.id} thumbnail connection status: ${connectionStatus}`);
 
+    // FIXME rename 'mutedWhileDisconnected' to 'mutedWhileNotRendering'
     // Update 'mutedWhileDisconnected' flag
-    this._figureOutMutedWhileDisconnected(!isActive);
-
-    if(this.connectionIndicator)
-        this.connectionIndicator.updateConnectionStatusIndicator(isActive);
+    this._figureOutMutedWhileDisconnected();
+    if(this.connectionIndicator) {
+        this.connectionIndicator.updateConnectionStatusIndicator(
+            connectionStatus);
+    }
 
+    const isInterrupted
+        = connectionStatus === ParticipantConnectionStatus.INTERRUPTED;
     // Toggle thumbnail video problem filter
     this.selectVideoElement().toggleClass(
-        "videoThumbnailProblemFilter", !isActive);
+        "videoThumbnailProblemFilter", isInterrupted);
     this.$avatar().toggleClass(
-        "videoThumbnailProblemFilter", !isActive);
+        "videoThumbnailProblemFilter", isInterrupted);
```

Comment:
It was intentional to not apply the grey filter for inactive status anymore, because the video was stopped by the bridge (it's a feature).